Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM elifesciences/php_cli
+
+USER elife
+RUN mkdir -p /srv/proofreader
+WORKDIR /srv/proofreader
+COPY composer.json /srv/proofreader/
+RUN composer install --classmap-authoritative --no-dev
+COPY . /srv/proofreader
+
+USER www-data
```

Comment:
Is that needed?

Reply:
Not that much, but it means by default if you this image to actually execute `proofreader` on some code, by mounting it inside it, it will run as `www-data` with minimum privilege. So far the main use case for this container is importing `proofreader` into another image to use it for tests.

Reply:
Wonder if there should be a read-only user for such purposes?